apps/01PigLatin.js

```diff
@@ -7,11 +7,87 @@ prompt.start();
 
 function pigLatin(word) {
 
-    // Your code here
+  // Concept Check
+  //
+  // Please explain why we are using String.prototype.toLowerCase() here.
+  // For more info:
+  // https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/toLowerCase
+  //
+  // Your Answer: We are using String.prototype.toLowerCase() to neutralize the
+  // user input and make it easier for our functions to analyze.
+  //
+  word  = word.toLowerCase();
 
+  // Concept Check
+  //
+  // What is the significance of setting vowelIndex to -1?
+  // This might help:
+  // https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/lastIndexOf
+  //
+  // Your Answer: Since an array starts with an index of 0, when we set the
+  // vowelIndex to -1, we are setting the index to start before the start of
+  // the array.
+  //
```

Comment:
@maddiesolz, **indexOf()** returns `-1` if something can't be found, so `-1` is of special significance here. We would not choose just any negative number.

Also note @mateoholman, we are working on strings here, not arrays. Though strings are array like.